addon/components/mobile/flexberry-objectlistview.js

```diff
@@ -138,4 +140,132 @@ export default FlexberryObjectlistview.extend({
     @default true
   */
   columnsWidthAutoresize: true,
+
+  /**
+    Array of objects corresponding to list of pages.
+
+    Each page is presented as object with following properties:
+    - **number** - Number of page.
+    - **isCurrent** - If `true` this page is current.
+    - **isEllipsis** - If `true` this page showing in pages list.
+
+    @private
+    @property _mobilePages
+    @type Array
+    @readOnly
+  */
+  _mobilePages: Ember.computed('pages', function() {
+    let mobilePages = Ember.A();
+    let pages = this.get('pages');
```

Comment:
Пусть это всегда будет эмберный массив.
```suggestion
    let pages = Ember.A(this.get('pages'));
```